serialmfg/serial_resources/dataset.py

```diff
@@ -28,6 +28,28 @@ def get(name, data_type, process_id=None):
         if serial.debug:
             print(f"Getting dataset with type {data_type}: {name}")
         return Dataset(client.make_api_request("/datasets", "GET", params=query_params)[0])
+    
+    @staticmethod
+    def create(name, data_type, process_id, extra_params=None):
```

Comment:
Do we use "extra_params" elsewhere? Is this a standard? i feel like "options" is more widely used

Reply:
It's not a standard, but they're not really options either. They're supplemental parameters that need/should be passed when creating numerical datasets